src/events.js

```diff
@@ -133,6 +133,10 @@ module.exports = (function () {
     return _handlers;
   };
 
+  _public.addEvents = function (events) {
+    allEvents.concat(events);
```

Comment:
lgtm throwing an error on this, says ignores result https://lgtm.com/rules/1510745846115/